### ISSUE TITLE
dev tools console logging gets suppressed by thimble's console

### ISSUE THREAD

**Issue by Pomax** [mozilla/thimble.mozilla.org#2186]
If I use `console.log` or `console.error` in my Thimble code now, the output ends up in the thimble console. That's useful for strings etc. but if I need to log objects so that I can inspect their content I still need to use the real dev tools, and it appears that the Thimble console is intercepting, rather than wrapping, the "real" console operations.

It might be necessary to borrow from jsbin here, where the on-site console panel shows console operations while still then sending them on to the `window.console. ... ` namespace.

Not sure if this is a true bug, but I'd consider it one.

**Comment by humphd** [comment]
I agree, we should be dual-logging vs. printing only to the in-browser one.  Also, I think we should consider using something like https://github.com/Automattic/util-inspect to better print objects.
